initial_scrape.py

```diff
@@ -0,0 +1,193 @@
+    time.sleep(3)
+    url1 = driver.current_url
+    print(url1)
+    driver.quit()
+    req1 = requests.get(url1)
+    soup1 = bs(req1.text, "lxml")
+    td1 = soup1.find_all("script")[3].contents[0]
+    le = td1.split(",")
+    d2, m = {}, []
+    for a in le:
+        if a.startswith('"/asset') and a not in m:
+            m.append(a)
+            indx = le.index(a)
+            d2[le[indx - 3]] = a[1:-1]
+    art = list(d2.keys())
+    length2 = len(art)
+    f1 = open("/Users/Lasitha/Documents/art_tweets/scraped_data/tweeted.txt", "r")
+    lines = f1.readlines()
+    for line in lines:
+        if typ in line:
+            oldline = line
+            line1 = line
+    ge1, ge2 = line1.index(":"), line1.index("\n")
+    if ge2 - ge1 == 1:
+        choiceindex = []
+    else:
+        choiceindex = line1[ge1 + 1 : ge2].split(" ")
+        for i in range(0, len(choiceindex)):
+            choiceindex[i] = int(choiceindex[i])
+
+    i = choice([i for i in range(0, length2) if i not in choiceindex])
+    url_for_art = d2[art[i]]
+    line1 = line1[:-1] + " " + str(i) + "\n"
+    new_lines = ""
+    for line in lines:
+        if oldline in line:
+            new_lines += line1
+        else:
+            new_lines += line
+    f1.close()
+    fe = open("/Users/Lasitha/Documents/art_tweets/scraped_data/tweeted.txt", "w")
+    fe.write(new_lines)
+    fe.close()
+    return [i, url_for_art, typ]
+
+
+def art_details():
+    b = scrape_out_arts()
+    print(b)
+    url_for_art = "https://artsandculture.google.com" + b[1] + "?hl=en"
+    print(url_for_art)
+    req4 = requests.get(url_for_art)
+    soup4 = bs(req4.text, "lxml")
+    im = soup4.find_all("img")[0]["src"]  # img_link
+    print(im)
+    td1 = soup4.find("section", class_="rw8Th QwmCXd")
+    td3 = td1.find_all("li")
+    d3 = {}
+    for i in range(len(td3)):
+        if "Title" in td3[i].find_all("span")[0].text:
+            ind = td3[i].text.index(": ")
+            k = td3[i].text[ind + 1 :][1:]
+            d3["Title"] = k
+        if "Creator" in td3[i].find_all("span")[0].text:
+            ind = td3[i].text.index(": ")
+            k = td3[i].text[ind + 1 :][1:]
+            d3["Creator"] = k
+        if "Date" in td3[i].find_all("span")[0].text:
+            ind = td3[i].text.index(": ")
+            k = td3[i].text[ind + 1 :][1:]
+            d3["Date"] = k
+        if "Location" in td3[i].find_all("span")[0].text:
+            ind = td3[i].text.index(": ")
+            k = td3[i].text[ind + 1 :][1:]
+            d3["Location"] = k
+        if "Medium" in td3[i].find_all("span")[0].text:
+            ind = td3[i].text.index(": ")
+            k = td3[i].text[ind + 1 :][1:]
+            d3["Medium"] = k
+        if "External Link" in td3[i].find_all("span")[0].text:
+            a = td3[i].find_all("a")
+            d3["External Link"] = a[0]["href"]
+    d3["Type"] = b[2]
+
+    f7 = open(
+        "/Users/Lasitha/Documents/art_tweets/scraped_data/current_tweet.txt", "w+"

```

Comment:
@LasithaE first of all, congrats on opening the PR! Two comments: next time onwards, please open PRs from a different branch. Also if you check link 179 (and other similar lines throughout), how do you plan to make sure the files exist on the user's PC?